.github/workflows/autofix.yml

```
@@ -44,18 +41,25 @@ jobs:
+              relative_path=$(realpath --relative-to="$(pwd)" "$file")
+              echo "$relative_path"
+            done | grep -q "^\.eslintrc\.json$|^\.eslintignore$"; then
             yarn lint:fix
           else
-            echo "$changed_files" | grep -E "(.js$|.jsx$|.ts$|.tsx$)" | xargs -r yarn lint:fix
-          fi
+            echo "$changed_files" | while read -r file; do
+              relative_path=$(realpath --relative-to="$(pwd)" "$file")
+              echo "$relative_path"
+            done | xargs -r yarn lint:fix
+          fi 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)